PostMetabox.php

```diff
@@ -292,7 +292,8 @@ protected function save( $post_id ) {
 			);
 			update_post_meta( $post_id, '_error_data_' . $this->id, $error_data );
 
-			$location = add_query_arg( 'message', 1, get_edit_post_link( $post_id, 'url' ) );
+			$location = esc_url_raw( add_query_arg( 'message', 1, get_edit_post_link( $post_id, 'url' ) ) );
+
 			wp_redirect( apply_filters( 'redirect_post_location', $location, $post_id ) );
```

Comment:
I get the thinking behind "let's make it super easy to see if we escaped", but there's a broader security guideline that says that it's best to escape as late as possible (i.e. closest to where the data is ultimately used). In this case, that would be right before the data is passed to `wp_redirect()`.

Come to think of it, I wonder why `wp_redirect()` doesn't do the escaping for us. :-|